config/v1/types_oauth.go

```diff
@@ -40,8 +40,31 @@ type OAuthSpec struct {
 	// templates allow you to customize pages like the login page.
 	// +optional
 	Templates OAuthTemplates `json:"templates"`
+	// audit specifies what should be audited in the context of OAuthServer.
+	// +optional
+	Audit OAuthAudit `json:"audit"`
+}
+
+// OAuthAudit specifies the Audit profile in use.
+type OAuthAudit struct {
+	// profile is a simple drop in profile type.
+	// +kubebuilder:default:="WriteLoginEvents"
+	Profile OAuthAuditProfileType `json:"profile,omitempty"`
 }
 
+// OAuthAuditProfileType is an enum that defines audit profiles.
```

Comment:
A common user does not know what an enum is.

`OAuthAuditProfileType specifies high-level configuration of the oauth-server audit profiles` maybe?

Reply:
Oh, you mean non-developers read that? Good to know the audience! thx